watchtower/wtdb/codec.go

```diff
@@ -0,0 +1,141 @@
+package wtdb
+
+import (
+	"io"
+
+	"github.com/lightningnetwork/lnd/channeldb"
+	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/watchtower/blob"
+	"github.com/lightningnetwork/lnd/watchtower/wtpolicy"
+)
+
+// UnknownElementType is an alias for channeldb.UnknownElementType.
+type UnknownElementType = channeldb.UnknownElementType
+
+// ReadElement deserializes a single element from the provided io.Reader.
+func ReadElement(r io.Reader, element interface{}) error {
```

Comment:
thanks! should help conserve a fair amount of code.

one thing that came to mind was to use the wt-specific types first, and only then fall back to the channeldb codec. in the end, i went this this ordering so that we could easily see, via coverage reports, any extraneous definitions in the wt-codec that could be removed. as things solidify, it may be useful to reverse this ordering